artemis/modules/joomla_scanner.py

```diff
@@ -47,6 +47,18 @@ def run(self, current_task: Task) -> None:
                 found_problems.append(f"Joomla version is too old: {joomla_version}")
```

Comment:
BTW, you may also iterate over some other xml files, as they also contain versions and might be present: not only joomla.xml, but also e.g. api/language/en-GB/install.xml or installation/language/en-GB/langmetadata.xml